### DISCUSSION TITLE
How do I change the default icon for native dialog?

### DISCUSSION THREAD

**Discussion by billywatkins** [tauri-apps/tauri#8616] [Q&A] [answered]
This is a noobie question, but how do I change the default folder icon for the native dialog on Mac? I went through the [dialog documentation](https://tauri.app/v1/api/js/dialog/), but I could not find any information.

This is my current preview:
<img width="323" alt="Screenshot 2024-01-16 at 10 54 01 AM" src="https://github.com/tauri-apps/tauri/assets/124872003/773c7bcf-d511-4933-87fd-02043a278be9">

**Comment by FabianLars** [comment]
As far as i know this is bound to the app icon on macOS which will only really works in the built app (after the `tauri build` command), did you try that already?

**Reply by billywatkins** [reply]
I understand that the build icon will only appear after it is built. How can I set a custom icon in the first place? Is there a Tauri config I can change?

**Reply by FabianLars** [reply, marked as answer]
It uses the app's main icon (so the same as in the dock etc) which i think tauri gets from the `icons` config in tauri.conf.json

**Reply by billywatkins** [reply]
Got it, thank you!
